src/client/controllers/email-sign-up-controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller } from 'stimulus';
+
+export default class EmailSignUpController extends Controller {
+  public processResponse(event) {
+    const [data] = event.detail;
```

Comment:
Where does the `detail` field come from?

Reply:
`rails-ujs`

Reply:
Could we add a short comment to say this controller uses `rails-ujs`, since the code doesn't have any references to `rails-ujs`?  I think it's clear the controller uses Stimulus for example since there's an import.

Reply:
Alternatively, is there a type for the UJS event?

Reply:
That's what I was thinking, I'll look into it

Reply:
No type sadly but it would be very easy to define myself—I think I'll go with that.